### PR TITLE
Updated language strings to be more understandable.

### PR DESCRIPTION
**Pull request by xtech86**

### CODE REVIEW

File: component/admin/language/en-GB.com_jevents.ini [hunk 1]
```diff
@@ -526,7 +526,7 @@ JEV_CALENDAR_IMPORT="Ical Import"
 JEV_TIMEZONE_INFO="Choose a valid Timezone value from <a href='http://www.php.net/manual/en/timezones.php' title='Timezone info' target='_blank'>here</a>"
 JEV_TIMEZONE="Timezone for your server<br/>(requires PHP 5.1+)"
 JEV_TIMEZONE_DESC="This is the timezone used for inputting events correctly during iCal import - this is very important when importing events from a UTC datasource or if our server is in a different DST regimen.<br/><br/>Enter the appropriate value EXACTLY as indicated via the link above.<br/><br/>This timezone is also used for determining the current time of day."
-JEV_IGNORE_EMBEDDED_CATEGORIES="Ignore embedded category names?"
+JEV_IGNORE_EMBEDDED_CATEGORIES="Replace categories with the fallback category?"
```
Comment (GeraintEdwards) [inline component/admin/language/en-GB.com_jevents.ini]: How about
"Override the imported categories with this category?"

File: component/admin/language/en-GB.com_jevents.ini [hunk 2]
```diff
@@ -1251,4 +1251,6 @@ JEV_ERROR_NOT_AUTH_CSS="You are not authorised to use the Custom CSS Tool. Only
 
 JEV_MODULE_CONFIG="Module Config"
 JEV_ORPHAN_EVENTS_DETECTED_AND_PLACED_IN_SPECIAL_ORPHAN_EVENT_CATEGORY_OR_CALENDAR="Orphan events have been detected and repaired.<br/><br/>These have been placed in a special category or calendar called 'Orphans' so that you can edit or remove them as required.<br/>Please note that this special orphan category and calendar is unpublished by default. <br/><br/>If orphan events have been found we apologise for any inconvenience caused."
-JEV_DUPLICATE_EXTENSION_WARNING="We have duplicate entries in the extensions table.  These are being cleaned up.  <br/><br/><strong>Please check your configuration settings and save them</strong>"
\ No newline at end of file
+JEV_DUPLICATE_EXTENSION_WARNING="We have duplicate entries in the extensions table.  These are being cleaned up.  <br/><br/><strong>Please check your configuration settings and save them</strong>"
+
+JEV_FALLBACK_CATEGORY="Fallback Category"
```
Comment (GeraintEdwards) [inline component/admin/language/en-GB.com_jevents.ini]: then
"Override Category"

I think 'fallback' is quite a subtle english word ???

File: component/admin/language/en-GB.com_jevents.ini (hunk 1 above)
Comment (xtech86) [inline component/admin/language/en-GB.com_jevents.ini]: But this is a yes/no field no field which says whether to use the selected category instead? 

You could have 

''Ignore imported categories and use the above selected category?'

File: component/admin/language/en-GB.com_jevents.ini (hunk 2 above)
Comment (xtech86) [inline component/admin/language/en-GB.com_jevents.ini]: I would then change this to then 

'Default / Override Category'

As it's used if there are no categories set too.

File: component/admin/language/en-GB.com_jevents.ini (hunk 1 above)
Comment (GeraintEdwards) [inline component/admin/language/en-GB.com_jevents.ini]: "Replace categories with the default category from above?"

File: component/admin/views/icals/tmpl/edit.php [hunk 3]
```diff
@@ -99,7 +99,7 @@ function submitbutton(pressbutton) {
 
         <div class="control-group">
             <div class="control-label">
-                <?php echo JText::_("Select_Default_Category"); ?>
+                <?php echo JText::_("JEV_FALLBACK_CATEGORY"); ?>
```
Comment (GeraintEdwards) [inline component/admin/views/icals/tmpl/edit.php]: I don't want us to use "fallback" - non-native english speakers will find it difficult.  I prefer "Default" and clarify the explanations if needed

**Pull request merged**